boot/bootutil/src/loader.c

```diff
@@ -690,18 +690,46 @@ split_image_check(struct image_header *app_hdr,
     return 0;
 }
 
+/*
+ *
+ */
 static inline int
-boot_magic_is_erased(uint8_t erased_val, uint32_t magic)
+boot_data_is_setto(uint8_t val, void *data, size_t len)
 {
     uint8_t i;
-    for (i = 0; i < sizeof(magic); i++) {
-        if (erased_val != *(((uint8_t *)&magic) + i)) {
+    uint8_t *p = (uint8_t *)data;
+    for (i = 0; i < len; i++) {
+        if (val != p[i]) {
             return 0;
         }
     }
     return 1;
 }
 
+static int
```

Comment:
Missing docs.

Reply:
This is a good point, but if I were to document, I would almost replicate the function's name. Also it uses the in memory boot data which is something that I wouldn't explain in a doc header so in the end this is too specific to bootloader itself, not an API of something that anyone would ever use.